package.json

```diff
@@ -67,9 +67,11 @@
   "dependencies": {
     "async": "^2.6.0",
     "boom": "^7.1.1",
+    "debug": "^3.1.0",
     "detect-node": "^2.0.3",
     "hapi": "^16.6.2",
     "hat": "0.0.3",
+    "ipfs-api": "^17.3.0",
```

Comment:
can we keep this as a devDependency? We don't want to bundle this, in case the consuming project already defines their own version.

Reply:
Sorry about that.  It seems that `npm install` just does that.